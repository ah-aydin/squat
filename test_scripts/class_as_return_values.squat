class Person {
    int age;
    int height;
    string name;
}

func build_person(int age, int height, string name) Person {
    Person person = Person(age, height, name);
    return person;
}

func main() {
    Person person1 = build_person(20, 177, "Gigel");
    println(person1.age);
    println(person1.height);
    println(person1.name);
}
