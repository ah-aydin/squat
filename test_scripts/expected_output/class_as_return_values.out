Compiling and running file: test_scripts/class_as_return_values.squat
20
177
Gigel
